### DISCUSSION TITLE
cannot find docker in linux build

### DISCUSSION THREAD

**Discussion by MrBitBucket** [pypa/cibuildwheel#851] [Q&A] [answered]
Not sure is this sort of thing is a normal kind of event, but I am seeing this error jn my ubuntu build part of

https://github.com/MrBitBucket/pyRXP-mirror/blob/master/.github/workflows/buildwheels.yml

```
Unable to find image 'quay.io/pypa/manylinux2010_x86_64:2021-09-19-a5ef179' locally
  Error response from daemon: Get "https://quay.io/v2/": dial tcp: lookup quay.io: No address associated with hostname
                                                              ✕ 0.28s
Error: Command ['docker', 'create', '--env=CIBUILDWHEEL', '--name=cibuildwheel-8b104a9b-9176-4d8b-9e66-dd0da1423a70', '--interactive', '--volume=/:/host', '-w', '/project', 'quay.io/pypa/manylinux2010_x86_64:2021-09-19-a5ef179', '/bin/bash'] failed with code 1. None
Error: Process completed with exit code 1.
```
The error is here https://github.com/MrBitBucket/pyRXP-mirror/runs/3753383166

 I ran a successful build yesterday in a different repo (which I used for debugging) https://github.com/MrBitBucket/dpyRXP/actions/runs/1286789038

**Comment by Czaki** [comment, marked as answer]
This is network error. Wait few hours and restart the building process.

**Reply by MrBitBucket** [reply]
So github / ms has network errors :-1:

**Reply by Czaki** [reply]
or docker image provider.
